styleguide/derek/_partials/_ceiling.ejs

```diff
@@ -5,13 +5,13 @@
           <nav class="ceiling-menuNav">
               <ul class="ceiling-menuInline" role="menu">
                 <li class="ceiling-menuInline-item">
-                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal">Call Us</a>
+                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal"><span class="hidden-xs">Call Us</span></a>
```

Comment:
Thinking @WidgetsBurritos is probably going to want this with a more maintainable class name huh?